engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleProcessStateDefinitionManager.scala

```diff
@@ -37,7 +38,8 @@ object SimpleProcessStateDefinitionManager extends ProcessStateDefinitionManager
     SimpleStateStatus.DuringCancel -> "/assets/states/stopping-running-animated.svg",
     SimpleStateStatus.Failed -> "/assets/states/failed.svg",
     SimpleStateStatus.Finished -> "/assets/states/success.svg",
-    SimpleStateStatus.Error -> "/assets/states/error.svg"
+    SimpleStateStatus.Error -> "/assets/states/error.svg",
+    SimpleStateStatus.RunningError -> "/assets/states/error.svg"
```

Comment:
Maybe we should use other icon to show that at the end this process is running like `deploy-failed.svg` for now and we should ask our ui-designers to add some `deploy-warning.svg`?

Reply:
You are right - I will wait for incons.